ga4gh/cli.py

```diff
@@ -933,6 +828,137 @@ def client_main(parser=None):
             # TODO suppress exception unless debug settings are enabled
             raise exception
 
+
+##############################################################################
+# ga2vcf
+##############################################################################
+
+class Ga2VcfRunner(SearchVariantsRunner):
+    """
+    Runner class for the ga2vcf
+    """
+    def __init__(self, args):
+        super(Ga2VcfRunner, self).__init__(args)
+        self._outputFile = args.outputFile
+        self._binaryOutput = False
+        if args.output_format in "bcf":
```

Comment:
The idea was to allow partial matching. However, this is a bit complicated (and I got it wrong in different places) so I've gone with the simpler exact match. Update pushed.